docusaurus/docs/manage/02-router/01-overview.md

```diff
@@ -0,0 +1,63 @@
+---
+id: router-deploying-overview
+title: Deployment Overview
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import styles from './styles.module.css';
+
+In this article, we are describing the process of router deployment and what attributes can be updated, removed, added after the deployment is completed.  
+
+## Components of Deployment
+
+:::info Notes
+Currently, the DNS name or IP address can not be changed after the deployment, because it requires to update the `Sans` section of the router server certificate. To do that one would need to re-enroll the router, which can not be done without creating a new router.
+:::
+
+<Tabs>
+<TabItem value="Identity" label="Identity" attributes={{className: styles.green}}>
+
+The pki-related fields in the `identity` section of the router configuration files are important to understand. These
+files are generated during the process of [enrolling](#enrollment) the router. These files do not need to exist before and will be created during the enrollment process. 
+This means the process running the enrollment will need the correct privileges granted in order to write - or overwrite those files in case of re-enrollment.  
+If the key specified in the identity section already exists - it will not be overwritten. Instead, it will be used during the enrollment process. In other words, you can create your own key or use existing key.
+
+</TabItem>
+<TabItem value="Config" label="Configuration File" attributes={{className: styles.orange}}>
+
+The router is configured using a [yaml](https://yaml.org/) file. The configuration file can be obtained by running the `ziti` CLI binary on the destiantion host, 
+but some configration options will need to be passeed to match the deployment details, i.e hostname, ip, private or public routers, etc. See `Process of deployment` for more.
+
+</TabItem>
+</Tabs>
+
+## Sizing Guidelines
+More details can be found [here](03-sizing.md)
+## Process of Deployment
+In order to eliminate sharing private key(s) over internet (i.e. existing key(s) located on a target host), it is best practice to run the cli router creation commands  on the target host. 
+
+:::info Note
+Ensure you are [logged in](../04-cli/02-logging-in.md) 
+for creating/udating routers through Cli
+:::
+
+1. ### Download ziti and ziti-router binaries

```

Comment:
I think headings alone, without the ordered list, are sufficient here to express the procedure. The formatting is awkward combining the list with heading.

Reply:
yes probably, thinking was to use heading to create sidebar shortcuts